.golangci.yaml

```diff
@@ -11,5 +9,8 @@ linters-settings:
     extra-rules: true
 linters:
   enable:
+    - unused
     - forbidigo
-    - unused
\ No newline at end of file
+issues:
+  exclude-dirs:
+    - vendor
```

Comment:
Why do we need this?

Reply:
vendor directory stores a copy of all the code the module depends on. vendor directory is usually large if present and we always want to skip it.

Reply:
I know but we don't vendor dependencies :)

Reply:
okay if we don't require i will remove it